tests/run/cpp_class_attrib.srctree

```diff
@@ -0,0 +1,83 @@
+# tag: cpp
+
+PYTHON setup.py build_ext --inplace
+PYTHON -c "import runner"
+
+######## setup.py ########
+
+from Cython.Build.Dependencies import cythonize
+from distutils.core import setup
+import os
+import pprint
+
+CYTHON_PROJECT_DIR = os.environ['CYTHON_PROJECT_DIR']
+example_dir = os.path.abspath(os.path.join(CYTHON_PROJECT_DIR,
+                              'docs/examples/userguide/wrapping_CPlusPlus'))
+
+ext_modules= cythonize("rect.pyx", include_path=[example_dir])
+ext_modules[0].include_dirs.append(example_dir)
```

Comment:
Is this really needed? It should be covered by the `include_path=` above, no?

Reply:
removed